### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by KristjanESPERANTO**
- wording
- Magic Mirror -> MagicMirror
- format

### CODE REVIEW

File: configuration/autostart.md [hunk 1]
```diff
@@ -197,7 +197,7 @@ sudo systemctl disable magicmirror.service
 Create file `/home/server/.config/lxsession/LXDE-pi/autostart` with the
 following contents:
 
-```
+```sh
```
Comment (rejas) [inline configuration/autostart.md:200]: is there a rule for when to use shel and when to use sh?

**Comment by sdetweil** [comment]
thats why i use pm2.  they figure it out.same on every platform

File: configuration/autostart.md (hunk 1 above)
Comment (KristjanESPERANTO) [reply configuration/autostart.md:200]: No official rule. For scripts I use `sh` and otherwise `shell` or `bash`. Should I use `shell` everywhere?

**Review by rejas — COMMENTED** [review]
...

File: configuration/autostart.md (hunk 1 above)
Comment (rejas) [reply configuration/autostart.md:200]: i see. so all other occurences in the docs are following this too then?

Comment (KristjanESPERANTO) [reply configuration/autostart.md]: I admit that I have not checked this in detail. I only noticed code blocks here without any indication of how they are to be rendered. 

I have just checked, so far only `shell` has been used.

Actually, the distinction as I have just explained it makes little sense, for the user it is displayed the same anyway. I just harmonised it.

**Review by rejas — APPROVED** [review]

**Pull request merged**